docs/messaging/azure-service-bus-component.md

```diff
@@ -180,12 +185,47 @@ The .NET Aspire Azure Service Bus component uses the following log categories:
 
 - `Azure.Core`
 - `Azure.Identity`
+- `Azure-Messaging-ServiceBus`
 
 ### Tracing
 
-The .NET Aspire Azure Service Bus component will emit the following tracing activities using OpenTelemetry:
-
-- "Azure.Data.Tables.TableServiceClient"
+> [!NOTE]
+> Service Bus `ActivitySource` support in the Azure SDK for .NET is experimental, and the shape of activities may change in the future without notice.
+
+You can enable tracing in one of three ways:
+
+- Set the `DisableTracing` option to `false` in the configuration.
```

Comment:
This won't do it. The default is `false`. So setting it to `false` won't do anything.